ledgerautosync/converter.py

```diff
@@ -194,13 +195,21 @@ def mk_dynamic_account(self, payee, exclude):
 
 class OfxConverter(Converter):
     def __init__(self, ofx, name, indent=4, ledger=None, fid=None,
-                 unknownaccount=None, payee_format=None):
+                 unknownaccount=None, payee_format=None, hardcodeaccount=None, shortenaccount=None):
         super(OfxConverter, self).__init__(ledger=ledger,
                                            indent=indent,
                                            unknownaccount=unknownaccount,
                                            currency=ofx.account.statement.currency,
                                            payee_format=payee_format)
-        self.acctid = ofx.account.account_id
+        if hardcodeaccount is not None:
+            self.real_acctid = ofx.account.account_id
+            self.acctid = hardcodeaccount
+        elif shortenaccount:
+            self.real_acctid = ofx.account.account_id
+            self.acctid = ofx.account.account_id[-4:]
+        else:
+            self.real_acctid = ofx.account.account_id
```

Comment:
Since this line is the same in all branches, you can place a single line outside the branches, above line 205